### PR TITLE
[ET-1410] Implement modal that triggers after a successful connection with Stripe.

### PR DESCRIPTION
**Pull request by juanfra**
### 🎫 Ticket

- [ET-1410] 

### 🗒️ Description

- Implementing a modal that opens after coming back from Stripe and having a successful connection.
- Text TBD: Most likely warning about webhooks configuration and payment methods/currency stuff (link to a Stripe primer).

### 🎥 Artifacts 

https://user-images.githubusercontent.com/252415/153714367-f36a2da9-6d05-4357-a249-b58f79d277d7.mov

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [ ] My code is tested. 
- [ ] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1410]: https://theeventscalendar.atlassian.net/browse/ET-1410?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete/notice-test-mode.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+tribe( 'tickets.editor.template' )->template(
+	'components/notice',
+	[
+		'id'             => 'tec-tickets__admin-settings-tickets-commerce-gateway-modal-notice-error',
+		'notice_classes' => [
+			'tribe-tickets__notice--error',
+			'tec-tickets__admin-settings-tickets-commerce-gateway-modal-notice-error',
+		],
+		'content'        => __( 'You have connected your account for test mode. While in test mode no live transactions are processed.', 'event-tickets' ),
```
Comment (moraleida) [inline src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete/notice-test-mode.php:23]:
```suggestion
		'content'        => __( 'Your connection is active, but Tickets Commerce is set to test mode. While in test mode no live transactions are processed.', 'event-tickets' ),
```

**Review by moraleida — CHANGES_REQUESTED** [review]

File: src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		'tribe-modal__title--gateway-connected',
+	],
+];
+
+ob_start();
+$dialog_view->render_modal( $content, $args, 'stripe-connected-modal-id' );
+$modal_content = ob_get_clean();
+
+$modal  = '<div class="tribe-common event-tickets">';
+$modal .= '<span id="' . esc_attr( 'stripe-connected-modal-target' ) . '"></span>';
```
Comment (codingmusician) [inline src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete.php:38]:
```suggestion
$modal .= '<span id="stripe-connected-modal-target"></span>';
```

**Review by codingmusician — DISMISSED** [review]

File: src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete/content.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		<?php
+			esc_html_e( 'Be sure that your Stripe currency matches the currency you have configured for Tickets Commerce, to avoid any issues or unexpected conversion fees.', 'event-tickets' );
+		?>
+	</div>
+
+	<div class="tec-tickets__admin-settings-tickets-commerce-stripe-modal-content-section">
+		<strong><?php esc_html_e( 'Payment methods', 'event-tickets' ); ?></strong> &mdash;
+		<?php
+		printf(
+			// Translators: %1$s: opening `a` tag to stripe's dashboard. %2$s: closing `a` tag to the knowledge base article.
```
Comment (moraleida) [inline src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete/content.php:26]:
```suggestion
			// Translators: %1$s: opening `a` tag to stripe's dashboard. %2$s: closing `a` tag.
```

File: src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete/content.php [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+			'</a>'
+		);
+		?>
+	</div>
+
+	<div class="tec-tickets__admin-settings-tickets-commerce-stripe-modal-content-section">
+		<strong><?php esc_html_e( 'Webhooks', 'event-tickets' ); ?></strong> &mdash;
+		<?php
+		printf(
+			// Translators: %1$s: opening `a` tag to the knowledge base article. %2$s: closing `a` tag to the knowledge base article.
```
Comment (moraleida) [inline src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete/content.php:38]:
```suggestion
			// Translators: %1$s: opening `a` tag to the knowledge base article. %2$s: closing `a` tag.
```

File: src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete/content.php [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+			'</a>'
+		);
+		?>
+	</div>
+
+	<div class="tec-tickets__admin-settings-tickets-commerce-stripe-modal-content-section">
+		<strong><?php esc_html_e( 'PCI Compliance', 'event-tickets' ); ?></strong> &mdash;
+		<?php
+		printf(
+			// Translators: %1$s: opening `a` tag to the knowledge base article. %2$s: closing `a` tag to the knowledge base article.
```
Comment (moraleida) [inline src/admin-views/settings/tickets-commerce/stripe/modal/signup-complete/content.php:50]:
```suggestion
			// Translators: %1$s: opening `a` tag to the knowledge base article. %2$s: closing `a` tag.
```

**Review by moraleida — APPROVED** [review]
minor fixes and we're :+1:

File: src/Tickets/Commerce/Gateways/Stripe/Settings.php [hunk 6]
```diff
@@ -195,9 +195,9 @@ public function get_settings() {
 		);
 
 		$checkout_type_tooltip = sprintf(
-		// Translators: %1$s: Opening `<a>` tag for Stripe docs link. %2$s: Closing `<a>` tag for Stripe docs link.
-			__( 'Additional payment methods are available based on currency and location and must be enabled individually within your Stripe account.  Learn more about Stripe checkout and payment configuration %1$shere%2$s.', 'event-tickets' ),
-			'<a href="https://stripe.com/docs/payments/checkout/web#checkout-payment-method-options" target="_blank" rel="noopener noreferrer">', // @todo: @juanfra: Update this link with our KB article.
+			// Translators: %1$s: Opening `<a>` tag for KB article link. %2$s: Closing `<a>` tag for KB article link.
```
Comment (juanfra) [inline src/Tickets/Commerce/Gateways/Stripe/Settings.php:198]:
```suggestion
			// Translators: %1$s: Opening `<a>` tag for KB article link. %2$s: Closing `<a>` tag.
```

**Review by moraleida — APPROVED** [review]

**Pull request merged**